src/Microsoft.ML.TensorFlow/TensorFlow/TensorflowUtils.cs

```diff
@@ -19,17 +19,6 @@ public static class TensorFlowUtils
         public const string OpType = "OpType";
         public const string InputOps = "InputOps";
 
-        // This method is needed for the Pipeline API, since ModuleCatalog does not load entry points that are located
-        // in assemblies that aren't directly used in the code. Users who want to use TensorFlow components will have to call
-        // TensorFlowUtils.Initialize() before creating the pipeline.
-        /// <summary>
-        /// Initialize the TensorFlow environment. Call this method before adding TensorFlow components to a learning pipeline.
-        /// </summary>
-        public static void Initialize()
-        {
-            ImageAnalytics.Initialize();
-        }
```

Comment:
What is the new alternative to this, for the sake of the legacy API?

Reply:
We only have 2 of these empty `Initialize()` methods, so it wasn't really a holistic plan.
If someone is using the "new API" this isn't necessary at all.

If someone is using the legacy API, and needs to do this, they can follow the same approach outlined in the following issues:

* https://github.com/dotnet/machinelearning/issues/569#issuecomment-409343579
* https://github.com/dotnet/machinelearning/issues/559

Basically, they can just get the `typeof` some type in that assembly to ensure it is loaded up front.